lib/src/main/java/grpcbridge/util/ProtoJson.java

```diff
@@ -42,8 +67,17 @@ public static RpcMessage parse(HttpResponse httpResponse, Message.Builder builde
     }
 
     public static HttpResponse serialize(RpcMessage message) {
-        String body = serialize(message.getBody());
-        return new HttpResponse(body, message.getMetadata());
+        String httpBody;
+        if (message.getMethodType().serverSendsOneMessage()) {
+            httpBody = !message.getBody().isEmpty() ? serialize(message.getBody().get(0)) : "";
+        } else {
+            List<String> messagesBody = new ArrayList<String>();
+            for (Message msg : message.getBody()) {
+                messagesBody.add(serialize(msg));
+            }
+            httpBody = "[" + Joiner.on(",").join(messagesBody) + "]";
```

Comment:
There is another joiner method that accepts prefix and suffix I think.

Reply:
Can also use StringBuilder here and avoid temporary messageBody list.

Reply:
I switched it to StringBuilder, thanks.